concepticondata/conceptlists/Zgraggen-1980-380.tsv

```diff
@@ -308,40 +308,40 @@ Zgraggen-1980-380-374	2374	DREAM	374	dream
 Zgraggen-1980-380-375	3033	SCARED	375	be afraid
 Zgraggen-1980-380-380	1847	SICK	380	be sick
 Zgraggen-1980-380-381	1215	THOU	381	you
-Zgraggen-1980-380-382			382	our-dual
+Zgraggen-1980-380-382	3778	OUR (DUAL)	382	our-dual
 Zgraggen-1980-380-383	2310	US (OBLIQUE CASE OF WE)	383	us
-Zgraggen-1980-380-384	1213	YOU	384	you-pl-o2p
+Zgraggen-1980-380-384	3777	YOU (MORE THAN TWO OBLIQUE CASE)	384	you-pl-o2p
 Zgraggen-1980-380-385	2279	NO OR NOT	385	no-not
 Zgraggen-1980-380-387			387	negation-marker
 Zgraggen-1980-380-388	2680	MACHETE	388	kunai
-Zgraggen-1980-380-389	2274	YOUR (PLURAL)	389	your-pl
+Zgraggen-1980-380-389	3781	YOUR (MORE THAN TWO)	389	your-pl
 Zgraggen-1980-380-390	970	COCONUT	390	coconut
 Zgraggen-1980-380-391	2111	YOU TWO	391	you-dual
 Zgraggen-1980-380-392	2046	APRON	392	tanket
 Zgraggen-1980-380-393	732	YOUR (SINGULAR)	393	your-sg
 Zgraggen-1980-380-394	2642	HE OR SHE	394	he-she
-Zgraggen-1980-380-395	2400	THEIR	395	theirs
-Zgraggen-1980-380-396			396	your-pl-dual
+Zgraggen-1980-380-395			395	theirs
+Zgraggen-1980-380-396	2274	YOUR (PLURAL)	396	your-pl-dual
```

Comment:
I'm not sure if this mapping is precise enough. YOUR (PLURAL) does not specify the number of persons addressed which is crucial in this case.

Reply:
That mapping is intentional - they also have your-dual, which clearly only refers to dual, so this gloss here seems to imply that it can refer to both dual and plural, which is what YOUR (PLURAL) means.

Reply:
I would propose correcting the concept list and then correcting the mapping.
The data as in the book at least clearly exists, whereas I don't know whether the glosses we have here were ever actually used.

Btw, we could nevertheless retain the new concepts I now added as we might need them in the future.